packages/mongodb/_dev/deploy/docker/docker-compose.yml

```
@@ -0,0 +1,22 @@
+version: '2.3'
+services:
+  mongodb:
+    build: .
+    ports:
+      - 27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: example
+    entrypoint: ["mongod", "--bind_ip_all", "--replSet", "rs0"]
+    command: ['--logpath', '/var/log/mongodb.log']
```

Would it be possible to use a volume for the logs? so logs of the real service can be used in system testing.
```suggestion
    command: ['--logpath', '/var/log/mongodb/mongodb.log']
    volumes:
      - ${SERVICE_LOGS_DIR}/mongodb:/var/log/mongodb
```

---

There is some read permission error while using service log so we are using sample logs but we will try it out.
Thanks